static/app/views/replays/replays.tsx

```diff
@@ -61,7 +63,7 @@ class Replays extends React.Component<Props> {
       <React.Fragment key={replay.id}>
         <Link
           to={`/organizations/${organization.slug}/replays/${generateEventSlug({
-            project: replay['project.name'],
```

Comment:
good change, looks like `project.name` wasn't part of `static/app/views/replays/types.tsx` before either.